roles/clamav/tasks/schedule_scan.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Schedule Daily ClamAV Scan 
+  become: true 
+  vars:
+    clamav_scan_exclude_options: "{{ ['--exclude-dir='] | product(clamav_scan_exclude_directories) | map('join') | join (' ') }}"
+    clamav_scan_options: "{{'--move=/quarantine' if  clamav_quarantine else ''}} {{'--remove' if  clamav_remove else ''}}"
+  cron: 
+    name: "Daily ClamAV Scan" 
+    job: >
+      docker exec {{ clamav_container_name }} clamscan -r {{clamav_scan_options }}  {{ clamav_scan_exclude_options }} /scanned 2>&1 | tee -a {{ clamav_log_path }}/clamav_scan.log
+    cron_file: clamav_scan_schedule 
+    state: "{{ 'present' if clamav_enable_auto_scan else 'absent' }}" 
+    minute: "*"  
+    hour: "{{ clamav_cron_hour }}"    
```

Comment:
I think that would cause the execution every minute in the matching hour. minute needs a specific value